SignalsIntegrationTests/SignalsIntegrationTests/ZeroOrderPolicyTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿using Domain;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SignalsIntegrationTests.Infrastructure;
+using System;
```

Comment:
Go to Tools -> Options -> C# -> Advanced (IIRC) and check the option for system usings to be first.